opal/util/event.c

```diff
@@ -47,6 +48,8 @@ int opal_event_register_params(void)
 
 #ifdef __APPLE__
     opal_event_module_include = "select";
+#elif defined(__linux__)
```

Comment:
We should really use the PLATFORM macros here, so:
```
#if defined(PLATFORM_OS_LINUX)
```

Reply:
Done. Also changed the __APPLE__ one to PLATFORM_OS_DARWIN.